src/libp2p/connectionsManager.ts

```diff
@@ -133,24 +136,33 @@ export class ConnectionsManager {
     this.libp2p.connectionManager.on('peer:disconnect', connection => {
       console.log('Disconnected from', connection.remotePeer.toB58String())
     })
-    await this.storage.init(this.libp2p, peerId)
-
+    
     return {
       address: this.localAddress,
       peerId: peerId.toB58String()
     }
   }
-
-  public subscribeForTopic = async (channelData: IChannelInfo, io: any) => {
-    await this.storage.subscribeForChannel(channelData.address, io, channelData)
+  
+  public subscribeForTopic = async (channelData: IChannelInfo) => {
+    await this.storage.subscribeForChannel(channelData.address, channelData)
+  }
+  
+  public initStorage = async (staticPeerId?: PeerId) => {
```

Comment:
Isn't it better to set peerId initializeNode as a member of a class and use it here? So it would be just one line
`await this.storage.init(this.libp2p, this.peerId)`

Reply:
done